bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetAuxiliaryHandler.java

```diff
@@ -0,0 +1,116 @@
+import org.openwebnet4j.message.Who;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link OpenWebNetAuxiliaryHandler} is responsible for handling Auxiliary (AUX) commands/messages
+ * for an OpenWebNet device.
+ * It extends the abstract {@link OpenWebNetThingHandler}.
+ *
+ * @author Massimo Valla - Initial contribution
+ * @author Giovanni Fabiani - Auxiliary message support
+ */
+@NonNullByDefault
+public class OpenWebNetAuxiliaryHandler extends OpenWebNetThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(OpenWebNetAuxiliaryHandler.class);
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = OpenWebNetBindingConstants.AUX_SUPPORTED_THING_TYPES;
+
+    public OpenWebNetAuxiliaryHandler(Thing thing) {
+        super(thing);
+    }
+
+    /**
+     * Handles Auxiliary command for a channel
+     *
+     * @param channel the channel
+     * @param command the Command
+     */
+    @Override
+    protected void handleChannelCommand(ChannelUID channel, Command command) {
+        logger.debug("handleAuxiliaryCommand() (command={} - channel={})", command, channel);
+        Where w = deviceWhere;
+        if (w != null) {
+            if (channel.getId().equals(CHANNEL_AUX)) {
+                if (command instanceof StringType) {
+                    updateStatus(ThingStatus.ONLINE);
+                    try {
+                        if (command.toString().equals(Auxiliary.WhatAuxiliary.ON.name())) {
+                            send(Auxiliary.requestTurnOn(w.value()));
+                        } else if (command.toString().equals(Auxiliary.WhatAuxiliary.OFF.name())) {
+                            send(Auxiliary.requestTurnOff(w.value()));
+                        }
+                    } catch (OWNException e) {
+                        logger.warn("Exception while processing command {}: {}", command, e.getMessage());
+                    }
+                } else {
+                    logger.warn("Unsupported Command {}", channel);
+                    updateStatus(ThingStatus.UNKNOWN);
+                }
+            } else {
+                logger.warn("Unsupported ChannelUID {}", channel);
+            }
+        }
+    }
+
+    @Override
+    protected void requestChannelState(ChannelUID channel) {
+        super.requestChannelState(channel);
+        logger.debug("requestChannelState() thingUID={} channel={}", thing.getUID(), channel.getId());
+        if (channel.getId().equals(CHANNEL_AUX)) {
+            ThingStatus ts = getThing().getStatus();
+            if (ThingStatus.ONLINE != ts && ThingStatus.REMOVING != ts && ThingStatus.REMOVED != ts) {
+                updateStatus(ThingStatus.ONLINE);
+            }
+        }
+    }
+
+    @Override
+    protected void refreshDevice(boolean refreshAll) {
+        /*
+         * NOTICE: It is not possible to refresh the status of a
+         * WHO=9 command (for alarm control use WHO=5 instead)
```

Comment:
I would leave empty requestChannelState method as well. It cannot  give nothing useful. Because AUX provides no return from the BUS. As far as I know.  So I  would comment similarly as above. OK for the rest.